packages/design-system-twrnc-preset/src/typography.ts

```diff
@@ -118,16 +118,16 @@ export const typographyTailwindConfig: TypographyTailwindConfigProps = {
     ],
   },
   fontFamily: {
-    'default-regular': 'Geist Regular',
-    'default-regular-italic': 'Geist Regular Italic',
-    'default-medium': 'Geist Medium',
-    'default-medium-italic': 'Geist Medium Italic',
-    'default-bold': 'Geist Bold',
-    'default-bold-italic': 'Geist Bold Italic',
-    'accent-regular': 'MM Sans Regular',
-    'accent-medium': 'MM Sans Medium',
-    'accent-bold': 'MM Sans Bold',
-    'hero-regular': 'MM Poly Regular',
+    'default-regular': 'Geist-Regular',
+    'default-regular-italic': 'Geist-RegularItalic',
+    'default-medium': 'Geist-Medium',
+    'default-medium-italic': 'Geist-MediumItalic',
+    'default-bold': 'Geist-Bold',
+    'default-bold-italic': 'Geist-BoldItalic',
```

Comment:
Looks like these may have been missed during the semi bold updates they should be using `Geist-SemiBold` and `Geist-SemiBoldItalic` I think.


```suggestion
    'default-bold': 'Geist-SemiBold',
    'default-bold-italic': 'Geist-SemiBoldItalic',
```

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/862/commits/5b7a994d79121bef721d784572603a29b309d3c0